tests/test_stat_result.py

```diff
@@ -24,3 +27,12 @@ def test_nothing_if_s(self):
         assert self.stat_result.formatted_output == (
             "Test X with Benjamini-Hochberg correction, P_val:2.000e-02 "
             "Stat=1.000e+00")
+
+    def test_is_significant(self):
+        assert self.stat_result.is_significant
+
+    def test_is_insignificant(self):
+        self.stat_result.corrected_significance = False
+
+        self.assertFalse(self.stat_result.is_significant)
+        self.assertFalse(self.stat_result_insignificant.is_significant)
```

Comment:
These could be split in two tests, one testing ensuring it works from the pvalue/alpha comparison, and the other with the corrected significance flag.

It would then make more sense that we are modifying the current `self.stat_result` for this, without the need for a comment.

Reply:
Will do this